### DISCUSSION TITLE
gpu/

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#86] [Announcements]
# gpu/

https://lulaoshi.info/gpu/

**Comment by wywangen** [comment]
鲁老师您好，请问可以咨询一个问题吗？我看网上的很多教程包括您写的文章里面，kernel函数都是比较简单的加法减乘除，要是我自己有一个比较复杂的函数f，kernel函数可以调用f吗? 我尝试过几次都有报错。

**Reply by wywangen** [reply]
准确的说应该是kernel函数不能调用另外一个自定义类K里面的自定义函数 K.f 吧
